target/linux/generic/pending-5.15/850-0016-PCI-aardvark-Add-support-for-PME-interrupts.patch

```diff
@@ -25,9 +25,9 @@ Signed-off-by: Marek Behún <kabel@kernel.org>
 
 --- a/drivers/pci/controller/pci-aardvark.c
 +++ b/drivers/pci/controller/pci-aardvark.c
-@@ -1480,6 +1480,18 @@ static void advk_pcie_handle_int(struct
```

Comment:
Please remove this patch it is also upstream, see https://github.com/gregkh/linux/commit/cf7ef0e3029e43f85b525dd49420d56792a4e61f

Reply:
This is not resolved

Reply:
Should be now